packages/lib/src/toolbar/controls/utils.ts

```diff
@@ -0,0 +1,27 @@
+import type { MiddlewareState } from '@floating-ui/react';
+
+export async function download(
+  url: URL | (() => Promise<URL | Blob>),
+  filename: string,
+) {
+  const urlOrBlob = url instanceof URL ? url : await url();
+
+  const anchor = document.createElement('a');
+  anchor.download = filename;
+  document.body.append(anchor);
+
+  anchor.href =
+    urlOrBlob instanceof Blob ? URL.createObjectURL(urlOrBlob) : urlOrBlob.href;
+  anchor.click();
+
+  URL.revokeObjectURL(anchor.href);
+  anchor.remove();
+}
+
+export const floatingMinWidth = {
```

Comment:
Upper case ?

Reply:
Hmm I see your point, but camel case matches the other Floating UI middlewares.